products/sle/main.pm

```diff
@@ -1027,6 +1027,39 @@ sub load_slenkins_tests {
     return 0;
 }
 
+sub load_hpc_cluster_tests {
+    # We need to boot *before* waiting for support server
+    # Because of TianoCore UEFI boot method
+    loadtest "boot/boot_to_desktop";
+
+    # Wait for support server to complete its initialization
+    # Inspired by HA test, but we can't reuse it as it is
+    loadtest "hpc/wait_support_server";
+
+    # Standard boot and configuration
+    # Tests are disabled just for the development
+    #loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
+    #loadtest "console/consoletest_setup";
```

Comment:
why we need console_setup at all ? currently we not using it and not having any issues because of this

Reply:
It is standard for all QAM openQA tests to properly setup console for tests.

Reply:
at least hostname module is called already during image creation https://openqa.suse.de/tests/1469414#step/hostname/1 so no point to call it again in each test scenario which using image 
Regarding consoletest_setup probably you also can just add it to create_hdd_textmode flow and remove it from here and also all QAM tests ?

Reply:
I was wrong about hostname we need it actually

Reply:
`hostname` is needed yes, but you remark about `consoletest_setup` is good, could be useful for HA too, I need to try this (could save us some time).
I think the best way to add it is at the end of the `create_hdd` flow, just before the shutdown, no?

Reply:
@asmorodskyi `consoletest_setup` also unifies console font, ensures that permissions are set on serial dev. If you think that it can be done in another better way, just create PR for it and discussion can continue there.